server/src/main/java/com/lunchtime/controllers/MenuItemDishController.java

```diff
@@ -20,7 +21,7 @@
 import com.lunchtime.models.MenuItemDish;
 
 @RestController
-@RequestMapping("/api/menuitemdish")
+@RequestMapping("/api/menuItemDish")
```

Comment:
Use lowercase in URL path

Reply:
Fixed client and server part.

Reply:
Use "Find in all project" for finding all paths

Reply:
Thanks, fixed.